### PR TITLE
Enter key directive

### PR DESCRIPTION
**Pull request by Jinpeiqi**
@rong4188 @Mooophy

### CODE REVIEW

**Comment by Jinpeiqi** [comment]
@rong4188 @Mooophy 	Frontend validation for create page.

File: MeetU/MeetU/Scripts/Controllers/Meetup/create.js [hunk 1]
```diff
@@ -2,7 +2,45 @@
     "use strict";
     angular
         .module('meetupModule')
-        .controller('MeetupCreateController', MeetupCreateController);
+        .controller('MeetupCreateController', MeetupCreateController)
+        .directive('myValidation', function () {
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Controllers/Meetup/create.js]: you should not put a directive here. should be /Scripts/Directives/validation.js

also, myValidation is incorrect, you can use validation or meetupValidation

File: MeetU/MeetU/Scripts/Views/Meetup/Create.html [hunk 2]
```diff
@@ -30,13 +53,13 @@
...
                 <input type="text" class="form-control meetup-create-input js-meetup-create-where" id="meetup-create-where"
-                       ng-model="meetupCreating.createParams.where" />
+                       ng-model="meetupCreating.createParams.where" ng-required="true"/>
             </div>
         </div>
 
         <div class="form-group meetup-form-group">
             <div class="col-md-offset-2 col-md-10">
-                <button class="btn btn-default meetup-create-submit-button" ng-click="meetupCreating.submitForm()">CREATE</button>
+                <button class="btn btn-default meetup-create-submit-button" ng-click="meetupCreating.submitForm()" ng-disabled="meetupCreateForm.$invalid" >CREATE</button>
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Views/Meetup/Create.html]: I have already removed these two buttons, can you update it?

File: MeetU/MeetU/Scripts/angular-confirm.js [hunk 3]
```diff
@@ -98,5 +98,26 @@
 
           }
         }
-      });
+      })
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/angular-confirm.js]: directive should not be in this file as well.

**Pull request merged**